### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: client/src/assets/i18n/de.po [hunk 1]
```diff
@@ -3506,6 +3506,14 @@ msgstr ""
 "Beachten Sie, dass das Passwort auf das neu generierte Passwort geändert "
 "wird."
 
+msgid ""
+"Note: The public access setting is deactivated for the organization. Please "
+"contact your admins or hosting providers to activate the setting."
+msgstr ""
+"Hinweis: Die Einstellung für den öffentlichen Zugang ist für diese "
+"Organisation deaktiviert. Bitte wenden Sie sich an Ihren Administrator oder "
```
Comment (Elblinator) [inline client/src/assets/i18n/de.po:3514]:
```suggestion
"Organisation deaktiviert. Bitte wenden Sie sich an Ihren Administrator*in oder "
```

File: client/src/assets/i18n/de.po [hunk 2]
```diff
@@ -164,6 +164,9 @@ msgstr "Design aktivieren"
 msgid "Activate public access"
 msgstr "Öffentlichen Zugang aktivieren"
 
+msgid "Activate text-based submitter field in motion creation"
+msgstr "Textbasiertes Antragstellerfeld in der Antragstellung aktivieren"
```
Comment (Elblinator) [inline client/src/assets/i18n/de.po:168]:
```suggestion
msgstr "Textbasiertes Antragsteller*infeld in der Antragstellung aktivieren"
```

File: client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts [hunk 3]
```diff
@@ -587,7 +587,8 @@ export const meetingSettings: SettingsGroup[] = fillInSettingsDefaults([
                     },
                     {
                         key: `motions_create_enable_additional_submitter_text`,
-                        label: _(`Enable text-based submitter in motion creation`),
+                        label: _(`Activate text-based submitter field in motion creation`),
+                        helpText: _(`Requires permission to manage motions`),
```
Comment (Elblinator) [inline client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts:591]:
```suggestion
                        helpText: _(`Requires permission to manage motion metadata`),
```
That you need motion.can_manage is a bug 
It is wanted that you can use this feature if you have motion.can_manage_metadata
Do you already want this corrected here or should this be corrected while fixing this Issue: 
https://github.com/OpenSlides/openslides-client/issues/4457

**Review by Elblinator — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts (hunk 3 above)
Comment (emanuelschuetze) [reply client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts:591]: we fix this later if #4457 is merged

File: client/src/assets/i18n/de.po (hunk 2 above)
Comment (emanuelschuetze) [reply client/src/assets/i18n/de.po]: Done

File: client/src/assets/i18n/de.po (hunk 1 above)
Comment (emanuelschuetze) [reply client/src/assets/i18n/de.po:3515]: Done

**Review by Elblinator — APPROVED** [review]

**Pull request merged**